draft-ietf-cdni-request-routing-extensions.xml

```diff
@@ -421,6 +547,31 @@
         </section>
 
        </section>
+
+       <section anchor="IANA.cdni.metadata.footprint.types" title="CDNI Metadata Footprint Types">
+          <t>As described in section 7.2 of <xref target="RFC8006"/>, the "CDNI Metadata Footprint Types" subregistry
+              was created within the "Content Delivery Network Interconnection (CDNI) Parameters" registry. The created
+              namespace defines the valid values for Footprint Object Types, and is already populated with the types
+              described in Section 4.2.2.2 of <xref target="RFC8006"/>.
+          </t>
+          <t>The following table defines a request for the registration of the additional footprint types
```

Comment:
"footprint types" -> "footprint type" (there is only one in this case)

Reply:
Fixing also line 105.
Note that I'm not fixing the section "headline". Only the embedded text

Reply:
@orifinkelman please ack